scripts/pre_commit_linter.py

```diff
@@ -2752,16 +2752,17 @@ def perform_all_lint_checks(self):
             self._check_directive_scope
         )
         self._check_dependencies()
-        extra_js_files_messages = self.process_manager['extra']
-        js_and_ts_component_messages = self.process_manager['component']
-        directive_scope_messages = self.process_manager['directive']
-        sorted_dependencies_messages = self.process_manager['sorted']
-        controller_dependency_messages = self.process_manager['line_breaks']
-
-        all_messages = (
-            common_messages + extra_js_files_messages +
-            js_and_ts_component_messages + directive_scope_messages +
-            sorted_dependencies_messages + controller_dependency_messages)
+        extra_js_files_messages = self.process_manager.get('extra', ())
```

Comment:
Why these changes? In what circumstances will there not be a value for the given keys?

Please add a comment to explain.